action-run.sh

```diff
@@ -14,6 +14,12 @@ if [ "${CI}" = "true" ]; then
   set -e
 fi
 
+## Freeing space since 14GB are not enough anymore
+df -h
+echo "Deleting android, dotnet, haskell, CodeQL, Python, swift to free up space"
+sudo rm -rf /usr/local/lib/android /usr/share/dotnet /usr/local/.ghcup /opt/hostedtoolcache/CodeQL /opt/hostedtoolcache/Python /usr/share/swift
+df -h
```

Comment:
Can we also have the information of the free inodes available? If we are only needing 12G as we discussed offline, the issue might be because of the number of files rather than the file size.

Reply:
done